pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -144,7 +144,7 @@ func (r *ReconcileWindowsMachineConfig) Reconcile(request reconcile.Request) (re
 			// We assume the credential path is `/etc/aws/credentials` mounted as a secret.
 			wkl.CloudCredentialsPath,
 			instance.Spec.AWS.CredentialAccountID,
-			"", "", instance.Spec.InstanceType,
+			"/tmp", "", instance.Spec.InstanceType,
```

Comment:
Is it guaranteed that `/tmp` always exists?

Reply:
As long as we control the image creation, yes.

Reply:
It also depends on where we are copying the operator binary to as the trackerfile creation happens at the place where the binary is getting copied to. That being said, we should be a bit cautious where we're writing this file as this can help in debugging purposes.(We are also having configmap which we can track though but it won't have information on the sg and ports being opened etc)